srcpkgs/libclc/template

```diff
@@ -0,0 +1,38 @@
+# Template file for 'libclc'
+pkgname=libclc
+version=11.0.0
+revision=1
+wrksrc="${pkgname}-${version}.src"
+hostmakedepends="cmake clang llvm python3 libedit-devel libffi-devel
+ ncurses-devel zlib-devel"
+short_desc="Open implementation of the OpenCL C programming language"
+maintainer="Jürgen Buchmüller <pullmoll@t-online.de>"
+license="BSD-3-Clause, MIT"
+homepage="https://libclc.llvm.org"
+distfiles="https://github.com/llvm/llvm-project/releases/download/llvmorg-${version}/libclc-${version}.src.tar.xz"
+checksum=5a0a2f67b85ede09b2f08c071c5b0b6b5ad56bfb00b39a22c369afcc613c42ee
+provides="libclc-${version}_${revision}"
```

Comment:
Why is this line necessary?

Reply:
Actually I don't know if it is. There is no library, just header files, so it's probably not needed at all.